content/v2.0/reference/flux/stdlib/built-in/transformations/join.md

```diff
@@ -136,3 +138,14 @@ join(
   on: ["_time", "host"]
 )
 ```
+
+## join() versus union()
+`join()` outputs new rows comprised of data from each of the joined streams of tables.
+`union()` does not modify the data in each row, but unifies separate streams of tables
```

Comment:
join() inputs and combines rows f specified tables and outputs all rows in one table?
union() inputs specified tables w/o combining rows and outputs tables in union?